AcademicHub.Common/CustomExceptions/Address.cs

```diff
@@ -2,6 +2,6 @@ namespace CustomExceptions;
 
 public static class Address
 {
-    public static string academichuburl { get; } = "https://academichub.net/";
+    public static string academichuburl { get; } = "https://academichub.netcheck/";
     // public static string academichuburl { get; } = "http://localhost:3000";
```

Comment:
drop commented code. No sense to keep it here

Reply:
Also, not sure that this correlates with global exceptions handling. So take sometime to prepare an idea how to improve this

Reply:
commented code needs to be there in order to not write it every time you want to check whether the code works on localhost